Conversion/MultiCopy.FCMacro

```diff
@@ -0,0 +1,1439 @@
+                                            tempStr5 = tempStr4[
+                                                tempStr4.index("|") + 1 :
+                                            ]
+                                            if len(tempStr5) == 0:
+                                                return False
+                                            if not tempStr5 in tags_list:
+                                                return False
+                                        else:
+                                            tempVar = tempStr4[
+                                                tempStr4.index("#") + 1 :
+                                            ]
+                                        if (
+                                            len(tempVar) != 0
+                                            and not tempVar.isnumeric()
+                                        ):
+                                            return False
+                                        if len(tempVar) != 0 and int(tempVar) != float(
+                                            tempVar
+                                        ):
+                                            return False
+                                        if (
+                                            len(tempVar) != 0
+                                            and int(tempVar) > MAX_PAD_DIGITS
+                                        ):
+                                            return False
+                                        tempStr3 = tempStr3.replace(
+                                            tempStr4 + "}", "", 1
+                                        )
+                                    else:
+                                        return False
+                                return True
+        return False
+
+    def _getFirstWord(self, lineOfText):
+        """
+        This function retrieves the first word in a text string.
+
+        Arguments
+        ----------
+        lineOfText: Text string containing a line of the inputted Paste Code.
+        """
+        resultStr = ""
+        charFound = False
+        for textElem in lineOfText:
+            if (
+                textElem == "\n"
+                or textElem == "\r"
+                or textElem == "\t"
+                or textElem == " "
+            ):
+                if charFound:
+                    break
+            else:
+                resultStr += textElem
+                charFound = True
+        return resultStr
+
+    def _validate(self):
+        """
+        This function performs syntactic validation of the inputted Paste Code when 
+        the user focusses out of the associated input text box. If the code is valid, 
+        the function returns a true; else, it returns a false.
+        """
+        self._disable()
+        tempVar, tab_n, isValid_, fromCodeCompleted, tags_list = 0, 0, False, False, []
```

Comment:
I guess not. I'll simplify them out.

Reply:
It's not really for simplification but such one-liners should be reserved for cases where it makes sense to define all variable at once because they are tied together.
One-line definitions are bad for version control because if you change one variable (either the value or the name), unrelated variables also appear in the diff.

Reply:
Okay, Mr. Galou. I get you now.